src/main/java/org/usfirst/frc/team1294/robot/RobotMap.java

```diff
@@ -10,4 +11,7 @@
     // following variables to use with your drivetrain subsystem.
     public static int leftMotor = 2;
```

Comment:
We can always make sure next meeting. Keep note of that because it might cause issues.